Main/src/main/java/com/main/acad/dao/SimpleChapterDao.java

```diff
@@ -155,5 +171,262 @@ public FileReader getInformstioAboutChildren(String name) {
             connectionPool.surrenderConnection(connection);
         }
     }
+
+    @Override
+    public boolean createNewChildChapter(String chapterName, String nameFile, String chapterText, String nameSubChapters) {
+        try {
+            nameFile = nameFile.trim() + ".html";
+            File directoryFile = new File("D:\\IT\\JavaBeginFiles\\" + chapterName + "\\" + nameFile.trim());
```

Comment:
Move prefix for files location to the constant, better to the properties file as it may differ from computer to computer (in which application run)